lib/Frontend/ArgsToFrontendOutputsConverter.cpp

```diff
@@ -528,3 +531,20 @@ SupplementaryOutputPathsComputer::readSupplementaryOutputFileMap() const {
 
   return outputPaths;
 }
+
+bool SupplementaryOutputPathsComputer::diagnoseMissingEntries(
+    const OutputFileMap &OFM, const StringRef supplementaryFileMapPath) const {
+  bool hadError = false;
+  (void)InputsAndOutputs.forEachInputProducingSupplementaryOutput(
```

Comment:
I'd rather have this be part of the existing loop. It's one thing to do lookups that might fail, but it's another thing to repeat existing lookups just to keep two bits of code separate.

Reply:
Sure! Folded in.